src/lib/MedStickerEncryptionV2.js

```diff
@@ -0,0 +1,85 @@
+import create from "./factory";
+import { arrayBufferToString, encode } from "./utilities";
+import { CHARLIE } from "./constants";
+
+const scrypt = create("scrypt");
+const gcm = create("AES-GCM");
+
+/**
+ *
```

Comment:
Sometimes functions dont need comments, sometimes they do. But they never need blank lines where the comment should be, so please decide which ones should be described and not. For the ones that not, remove the blank lines